source/docs/zero-to-robot/step-2/python-setup.rst

```diff
@@ -0,0 +1,87 @@
+Python Installation Guide
+=========================
+
+This guide is intended for Python teams. LabVIEW teams can skip to :doc:`labview-setup` and Java/C++ teams can skip to :doc:`wpilib-setup`.
+
+From now on, the Python version will be referred to as :guilabel:`RobotPy`
+
+Prerequisites
+-------------
+
+Supported Operating Systems and Architectures:
+ * Windows 10 or higher, 32 bit or 64 bit. Arm version also supported
+ * Ubuntu 22.04 or higher, 64 bit. Fedora 36+ and Arch Linux are also known to work
+ * macOS 11 or higher
+
+Python 3.7 or higher must be installed
+ * `Python for Windows <https://www.python.org/downloads/windows/>`__
+ * `Python for macOS <https://www.python.org/downloads/macos/>`__
+ * Python should already be installed for Linux users
+
+Updating the RobotPy version will overwrite the previously installed one. Consider using `Virtualenv <https://virtualenv.pypa.io/en/latest/index.html>`__ if you want to keep distinct versions.
```

Comment:
I don't think we'd have any particular reason to recommend virtualenv over venv. The Python Packaging Authority recommend using venv: https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/